traveler-ios-ui/TravelerKitUI/Storyboards/BookingItem.storyboard

```diff
@@ -112,7 +112,8 @@
                                                         <rect key="frame" x="16" y="485" width="343" height="30"/>
                                                         <state key="normal" title="Terms and conditions"/>
                                                         <connections>
-                                                            <segue destination="dYu-Mg-MyD" kind="presentation" identifier="termsAndConditionsSegue" id="SDz-0D-BOA"/>
+                                                            <action selector="didSelectTermsAndConditions:" destination="2qR-Lh-IWg" eventType="touchUpInside" id="zY2-1n-D73"/>
```

Comment:
Did you mean to do this?

Reply:
Ops, it was introduced when I did rebasing. Fixed.